src/Executor.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // Executor This struct ensures the parallel execution of all command executions
 type Executor struct {
-	planets []Planet
+	planets []*Planet
```

Comment:
Actually, I do think it's a goroutine issue. structs passed on tu functions in go are AFAIK always called by reference, but before going into the goroutine in the for loop I got to create a new copy of planet to avoid "closure" in the goroutine
`
for _, planet := range executor.planets {
		// to avoid closure over the value planet. seems odd but it is recommended
		planet := planet
`
The code wont compile without hit planet := planet, but if the planet isn't a pointer to an object, changes made to that new planet wouldn't be changed in the original planet